test/integration/components/go_otel/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Build the testserver binary
+# Docker command must be invoked from the projec root directory
+FROM golang:1.22 AS builder
```

Comment:
since this is still tagged as WIP, perhaps switch this to 1.23 already given https://github.com/grafana/beyla/pull/1105 is also coming

Reply:
It anyway doesn't matter, as even if Beyla is compiled with 1.23, it should support components compiled in earlier versions.